_python_source_roots.py

```
@@ -0,0 +1,3 @@
+from tests._python_source_roots import PYTHON_SOURCE_ROOTS, REPO_ROOT
+
+__all__ = ["REPO_ROOT", "PYTHON_SOURCE_ROOTS"]
```

Ruff (0.15.11)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)